### ISSUE TITLE
Pin messages on top of a room!

### ISSUE THREAD

**Issue by albjeremias** [dino/dino#740]
yes, this would be an awesome picture.. i wonder if there exists a XEP.

**Comment by Echolon** [comment]
Duplicate of 

**Allow pinning chats to the top of the chat list** #290

I think it can be closed @fiaxh @mar-v-in

**Comment by selurvedu** [comment]
No, this is not a duplicate. That other issue is about pinning the chats in a chat list, this one is about pinning the messages in a chat.

**Comment by raspbeguy** [comment]
No, it is really a duplicate.
